client/src/pages/Dashboard/UserDetails/UserDetails.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import Card from '@mui/material/Card';
+import CardMedia from '@mui/material/CardMedia';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import Divider from '@mui/material/Divider';
+import useStyles from './useStyles';
+import { Box } from '@mui/material';
+import Rating from '@mui/material/Rating';
+import StarIcon from '@mui/icons-material/Star';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+interface UserDetailsProps {
+  user: {
+    id: number;
+    image: string;
+    name: string;
+    occupation: string;
+    rating: number;
+    description: string;
+    rate: number;
+    city: string;
+  };
+}
+
+const UserDetails: React.FC<UserDetailsProps> = ({ user }) => {
+  const { image, name, occupation, rating, description, rate, city } = user;
+
+  const classes = useStyles();
+  let ratingStar = 0;
+  if (rating === 5) {
+    ratingStar = 5;
+  } else if (rating === 4) {
+    ratingStar = 4;
+  } else if (rating === 3) {
+    ratingStar = 3;
+  } else if (rating === 2) {
+    ratingStar = 2;
+  } else if (rating === 1) {
+    ratingStar = 1;
+  } else {
+    ratingStar === ratingStar;
+  }
```

Comment:
This doesn't really make sense. Just assign `ratingStar` to the `rating`

Reply:
Better yet - we could just use the `rating` itself and save ourself a variable.

Reply:
Maybe I just overthink this and come up with this solution, rather than thinking in a simple way.